### PR TITLE
Missing default panel

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Review by northwestwitch — APPROVED** [review]
It works fine! 👍 

I've reproduced the error by removing the "is_default" key from the panel in test case. Master branch crashes. When I switch to this branch the variants are shown.

**Comment by dnil** [comment]
> It works fine! 👍
> 
> I've reproduced the error by removing the "is_default" key from the panel in test case. Master branch crashes. When I switch to this branch the variants are shown.

Brilliant! Thanks!

File: scout/server/blueprints/variants/controllers.py [hunk 1]
```diff
@@ -713,7 +713,9 @@ def case_default_panels(case_obj):
         case_panels(list): a list of panels (panel_name)
     """
     case_panels = [
-        panel["panel_name"] for panel in case_obj.get("panels", []) if panel["is_default"] is True
+        panel["panel_name"]
+        for panel in case_obj.get("panels", [])
+        if panel.get("is_default", None) is True
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/controllers.py:718]: You could also write if panel.get("is_default",False) is True, but the result would be the same

Comment (dnil) [reply scout/server/blueprints/variants/controllers.py:718]: Yes, I was torn.. 😸

**Pull request merged**